regex-syntax/src/hir/literal.rs

```diff
@@ -2455,6 +2455,22 @@ mod tests {
         assert_eq!(expected, e(r"^aZ*b"));
     }
 
+    #[test]
+    #[ignore = "Missing parser support for lookaround"]
+    fn lookaround() {
+        assert_eq!(exact(["ab"]), e(r"a(?<=qwe)b"));
```

Comment:
I know that's not the point of this test but this regex could never match right?

Reply:
Good point. Fixed